src/browser/locales/en-US/browser/welcome.ftl

```diff
@@ -17,3 +17,9 @@ welcome-dialog-theme-action = Continue
 
 welcome-dialog-search = Chose a search engine
 welcome-dialog-search-action = Done
+
+# About Welcome Page Dialog
```

Comment:
It is not a dialog. This comment is garbage

Reply:
Removed Dialog from code comment